spec/post_console_spec.rb

```diff
@@ -1,15 +1,15 @@
 require 'spec_helper'
 
 describe ".env" do
-  it 'should exist in root' do
-    expect(File).to be_file(".env")
+  it 'should exist in install' do
+    expect(File).to be_file("lib/generators/slacked/install/templates/.env")
```

Comment:
Put the path in a variable like:

let(:template_path){'lib/generators/slacked/install/templates/'}